jbot/src/main/java/me/ramswaroop/jbot/core/slack/models/Event.java

```diff
@@ -62,6 +65,8 @@
     private String domain;
     @JsonProperty("email_domain")
     private String emailDomain;
+    @JsonProperty("team")
+    private String teamId;
```

Comment:
Yes! what we actually get in response is team id, but slack named the property as team, which can very well be confused with [this](https://api.slack.com/methods/team.info) were "team" is a json object. Wish slack would have followed the conventions!!